app/src/screens/MapScreen.js

```diff
@@ -245,18 +286,26 @@ function MapScreen(): React.Node {
           getIcon: (_d: Point) => 'marker',
           sizeScale: 2,
           getPosition: d => [d.latitude, d.longitude, d.height],
-          getSize: d => 15,
+          getSize: d => 10,
           getColor: d => {
-            const red = parseInt(
-              255 - (minRssiToDisplay - d.rssi) * rangeFactor,
-            );
-            const blue = 255 - red;
-            const green = 255 - red - blue;
-            return [red, green, blue, 255];
-          },
-          updateTriggers: {
-            // This tells deck.gl to recalculate color when `rangeFactor` changes
-            getColor: rangeFactor,
+            if (d.rssi > -60) {
+              return getRGB(COLORS.heat10, COLORS.heat20, (60 + d.rssi) / 5);
+            } else if (d.rssi > -65) {
+              return getRGB(COLORS.heat20, COLORS.heat30, (65 + d.rssi) / 5);
+            } else if (d.rssi > -70) {
+              return getRGB(COLORS.heat30, COLORS.heat40, (70 + d.rssi) / 5);
+            } else if (d.rssi > -75) {
+              return getRGB(COLORS.heat40, COLORS.heat50, (75 + d.rssi) / 5);
+            } else if (d.rssi > -80) {
+              return getRGB(COLORS.heat50, COLORS.heat60, (80 + d.rssi) / 5);
+            } else if (d.rssi > -85) {
+              return getRGB(COLORS.heat60, COLORS.heat70, (85 + d.rssi) / 5);
+            } else if (d.rssi > -90) {
+              return getRGB(COLORS.heat70, COLORS.heat80, (90 + d.rssi) / 5);
+            } else if (d.rssi > -95) {
+              return getRGB(COLORS.heat80, COLORS.heat90, (95 + d.rssi) / 5);
+            }
+            return getRGB(COLORS.heat90, COLORS.heat100, (100 + d.rssi) / 5);
```

Comment:
Can we pull this out into its own function (in the utility file), and have it do this automatically from the `signalBuckets` variable?
Seems straightforward for looping